doc/developer/Building_FRR_on_CentOS6.rst

```diff
@@ -7,6 +7,16 @@ in redhat/README.rpm\_build.md on how to build a rpm package)
 
 Instructions are tested with ``CentOS 6.8`` on ``x86_64`` platform
 
+Warning:
+--------
+``CentOS 6`` is very old and not fully supported by the FRR community
+anymore. Building FRR takes multiple manual steps to update the build
```

Comment:
What does this sentence mean?  In what way is it not 'fully supported'?

Reply:
Please see the mailing list discussion regarding EOL platform support that you participated in.

Reply:
okay, but whatever definition is agreed to, that definition should be document *somewhere* - could be here (which is what I was suggesting) or a pointer to what EOL means...